tools/env/dev.ts

```diff
@@ -4,10 +4,10 @@ const DevConfig: EnvConfig = {
   ENV: 'DEV',
   RMMAPI: 'https://testdata.nist.gov/rmm/',
   SDPAPI: 'https://testdata.nist.gov/sdp/',
-  PDRAPI: 'https://testdata.nist.gov/od/id/',
+  PDRAPI: 'https://testdata.nist.gov/pdr/',
   DISTAPI: 'https://testdata.nist.gov/od/',
   METAPI: 'http://datapubtest.nist.gov/midas/',
-  LANDING: 'external'
+  LANDING: 'https://testdata.nist.gov/rmm/'
```

Comment:
@haffonist we do not use these values. As when you bring project up using docker, the env variables are passed at runtime. So even if you put xyz here does not matter.